plutus-core-spec/plutus-core-specification.tex

```diff
@@ -1,12 +1,13 @@
 % Plutus Core Specification
 \title{Formal Specification of\\the Plutus Core Language (version 2.0) }
-\date{22nd July 2019}
+\date{5th November 2019}
```

Comment:
Can we just drop these dates? I think they add nothing, and they're irregularly updated.